docs/content/s3.md

```diff
@@ -678,22 +678,6 @@ Specify if using an IBM COS On Premise.
 
 #### --s3-endpoint
```

Comment:
Can you drop the docs from the commit - I'll rebuild them when the release is made

Reply:
Done!